librz/core/cmd_linux_heap_glibc.c

```diff
@@ -50,4 +50,32 @@ RZ_IPI int rz_cmd_heap_fastbins_print(void *data, const char *input) {
 
 RZ_IPI RzCmdStatus rz_cmd_heap_arena_bins_print_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
 	call_handler(rz_cmd_heap_arena_bins_print_handler, argc, argv, mode);
+}
+
+/* API functions for Heap Viewer in Cutter */
```

Comment:
why are they in `cmd_linux_heap_glibc` if they are not commands?

Reply:
I cannot put them in `linux_heap_glibc.c` as they are not supposed to wrapped in the `GH` macro which is used across that file. Also, `cmd_linux_heap_glibc` uses the `call_handler` macro which I needed for these API  calls too. Do you have any other suggestions where I should put these?

Reply:
Mmmm I'm not sure.... let it be for now!